Cesium3DTilesSelection/include/Cesium3DTilesSelection/BingMapsRasterOverlay.h

```diff
@@ -2,8 +2,10 @@
 
 #include "Cesium3DTilesSelection/Library.h"
 #include "Cesium3DTilesSelection/RasterOverlay.h"
```

Comment:
I think this (unchanged) line is incorrect. Quoted includes are meant to be relative to the including file, right? So from here in `Cesium3DTilesSelection/include/Cesium3DTilesSelection/BingMapsRasterOverlay.h`, a relative path of `Cesium3DTilesSelection/RasterOverlay.h` would resolve to `Cesium3DTilesSelection/include/Cesium3DTilesSelection/Cesium3DTilesSelection/RasterOverlay.h`. It works, of course, because all of our compiler also resolve quoted includes using the include search path. But if we're aiming to use relative paths for in-library includes, it would be:

```
#include "RasterOverlay.h"
```